src/DynamoCoreWpf/ViewModels/Watch3D/DefaultWatch3DViewModel.cs

```diff
@@ -439,6 +439,12 @@ private void OnNodeRemovedFromWorkspace(NodeModel node)
             DeleteGeometryForIdentifier(node.AstIdentifierBase);
         }
 
+        public virtual CameraData GetCameraInformation()
+        {
+            // Override in derived classes.
+            return null;
+        }
```

Comment:
`DefaultWatch3DViewModel` is not abstract class, so it doesn't allow abstract members